proto/private/rules/proto_descriptor_set.bzl

```diff
@@ -41,20 +41,31 @@ def _proto_descriptor_set_impl(ctx):
     )
 
     return [
+        ctx.attr.internal_export_lib[ProtoInfo],
         DefaultInfo(
             files = depset([output]),
             runfiles = ctx.runfiles(files = [output]),
         ),
     ]
 
-proto_descriptor_set = rule(
+_proto_descriptor_set = rule(
     implementation = _proto_descriptor_set_impl,
     attrs = {
         "deps": attr.label_list(
             mandatory = False,
             providers = [ProtoInfo],
             doc = """
 Sequence of `ProtoInfo`s to collect `FileDescriptorSet`s from.
+""".strip(),
+        ),
+        "internal_export_lib": attr.label(
+            mandatory = True,
+            providers = [ProtoInfo],
+            doc = """
```

Comment:
This looks like a bad merge.

Reply:
I'm not sure I follow. Can you elaborate please?